qutip/core/states.py

```diff
@@ -955,9 +958,8 @@ def enr_fock(dims, excitations, state, *, dtype=_data.Dense):
 
     Parameters
     ----------
-    dims : list
-        A list of the dimensions of each subsystem of a composite quantum
-        system.
+    dims : list or array
+        The quantum state dimensions array, as it would appear in a Qobj.
```

Comment:
All `enr_...` functions take the same `dims` input. It's doc-string should match between the enr_operators and enr_states.